src/ducks/apps/components/ApplicationPage/Gallery.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+
+import { translate } from 'cozy-ui/react/I18n'
+
+export const Gallery = ({t, slug, images}) => {
+  return (
+    <div className='sto-app-images'>
+      {images && !!images.length &&
+        images.map((screen, index) =>
+          <img className='sto-app-image' src={screen} alt={`${slug}-image-${index + 1}`} />
```

Comment:
An alt attribute should give information about the content of the image, it should not be a technical name. If you don't have informations, keep the alt empty.
Otherwise we can required an alt attributes in the screenshots config in the manifest

Reply:
What if I use `${slug}-screenshot-${index+1}`? At least to say this is one of the application screenshots.